kubectl_plugins/bind-service/bind-service.go

```diff
@@ -0,0 +1,104 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"strings"
+
+	"github.com/kubernetes-incubator/service-catalog/kubectl_plugins/utils"
+	v1alpha1 "github.com/kubernetes-incubator/service-catalog/pkg/apis/servicecatalog/v1alpha1"
+	clientset "github.com/kubernetes-incubator/service-catalog/pkg/client/clientset_generated/clientset"
+
+	"k8s.io/client-go/pkg/api/v1"
+	"k8s.io/client-go/rest"
+)
+
+const USAGE = `
+
+Usage:
+  kubectl plugin bind-service SERVICE_CLASS_NAME SECRET_NAME NAMESPACE/INSTANCE_NAME
```

Comment:
namespace is a function of the entire binding, not just the instance name.  It's preferable to use `--namespace` for the whole operation.

Also, this operation should not need to take the service class at all.

Suggest the following order for arguments:

`bind-service INSTANCE_NAME SECRET_NAME --namespace <namespace>`